doc/installation.rst

```diff
@@ -18,56 +18,62 @@ Linux
 ~~~~~
 
 We provide pre-built binaries for some platforms from the Git ``master``
-branch. Artifacts are distributed as TAR archives which can be unpacked to any
-location::
+branch. Artifacts are distributed as DEB or RPM distribution packages, or TAR archives; TAR archives need to be unpacked to `/opt/spicy`::
 
-    # Unpack archive to e.g., /opt/spicy overwriting any previous installation.
-    # Writing to /opt/spicy likely requires superuser priviledges.
-    # The path /opt/spicy can be replaced with a custom path.
+    # Using DEB distribution packages.
+    dpkg --install spicy.deb
+
+    # Using RPM distribution packages.
+    rpm -i spicy.rpm
+
+    # Using TAR archives.
+    #
+    # Unpack archive to /opt/spicy (required) replacing any previous
+    # installation. Writing to /opt/spicy likely requires superuser privileges.
     rm -rf /opt/spicy && mkdir /opt/spicy
-    tar xf spicy-linux.tar.gz -C /opt/spicy --strip-components=1
+    tar xf spicy.tar.gz -C /opt/spicy --strip-components=1
 
 The binaries might require installation of additional dependencies;
 see the ``Dockerfile`` for the respective platform.
 
 alpine-3.12
```

Comment:
I keep thinking that it would be nice to turn all the download links into a real table. And then maybe have two columns "release" and "development" with corresponding links ("release" filled once we cut 1.0.0).